core-api/src/test/java/com/optimizely/ab/event/internal/EventBuilderV2Test.java

```diff
@@ -90,6 +90,7 @@ public void createImpressionEvent() throws Exception {
         assertThat(impression.getVisitorId(), is(userId));
         assertThat((double)impression.getTimestamp(), closeTo((double)System.currentTimeMillis(), 60.0));
         assertFalse(impression.getIsGlobalHoldback());
+        assertThat(impression.getAnonymizeIP(), is(projectConfig.getAnonymizeIP()));
```

Comment:
I don't think testing for that is needed. This tests that events are built correctly and not the fact that IPAnon should be `false` by default as that should be tested elsewhere. We should just make sure that the event value corresponds to what the project config shows. That said, I'll add a test for this case on the `ProjectConfigTest`